src/app/utils/undo-redo-history.ts

```diff
@@ -94,6 +95,9 @@ export function useHistory<S>(initialState: S): {
   const undo = useCallback(() => dispatch({ type: 'undo' }), []);
   const redo = useCallback(() => dispatch({ type: 'redo' }), []);
 
+  useHotkey('mod+z', '', undo);
+  useHotkey('mod+shift+z', '', redo);
+
```

Comment:
Hm, this hook is used by both Loadout Optimizer and the Loadout Drawer, which means they're fighting over this hotkey a bit (e.g. hotkey works on LO page, then click "save loadout" to bring up Loadout Drawer, now the hotkey works there, then close the sheet, now the hotkey doesn't work on the LO page).

Reply:
Hmm yeah. We really need this to have the semantics of "pushHotkey" where it can take precedence while the hook is in scope.

Reply:
That'd fix the "esc" behavior too.

Reply:
And help with #5286